browser/model/virtualbox.js

```diff
@@ -64,9 +64,7 @@ class VirtualBoxInstall extends InstallableItem {
                 resolve(output);
             });
           } else {
-            return new Promise((resolve, reject) => {
```

Comment:
I see the difference now, that one was inside that call. Current one is already inside Promise instance. You are right.